cluster/gke/util.sh

```diff
@@ -303,15 +305,20 @@ function detect-node-names {
 #   CLUSTER_NAME
 # Vars set:
 #   NODE_INSTANCE_GROUPS
+#   NODE_INSTANCE_GROUPS_URLS
 function detect-node-instance-groups {
   echo "... in gke:detect-node-instance-groups()" >&2
   local urls=$("${GCLOUD}" ${CMD_GROUP:-} container clusters describe \
     --project="${PROJECT}" --zone="${ZONE}" \
     --format='value(instanceGroupUrls)' "${CLUSTER_NAME}")
   urls=(${urls//;/ })
+  NODE_INSTANCE_GROUPS_URLS=${urls[*]}
   NODE_INSTANCE_GROUPS=()
   for url in "${urls[@]:-}"; do
-    NODE_INSTANCE_GROUPS+=("${url##*/}")
+    local instance_url=$(expr match ${url} '.*/zones/\([a-z0-9-]*\)/')
```

Comment:
`instance_url` seems like a weird name for this given that it's more like the zone of the MIG?